Sources/BraintreeCoreSwift/BTHTTP.swift

```diff
@@ -0,0 +1,542 @@
+import Foundation
+import Security
+
+/// Performs HTTP methods on the Braintree Client API
+// TODO: once BTAPIHTTP + BTGraphQLHTTP are converted this can be internal + more Swift-y
+// TODO: When BTAPIHTTP + BTGraphQL are converted we should update the dictionaries to [String: Any]
+@objcMembers public class BTHTTPSwift: NSObject, NSCopying, URLSessionDelegate {
+// TODO: - Mark interval vs private properties accordingly
+    public typealias RequestCompletion = (BTJSON?, HTTPURLResponse?, Error?) -> Void
+
+    // MARK: - Public Properties
+    /// An array of pinned certificates, each an NSData instance consisting of DER encoded x509 certificates
+    public let pinnedCertificates: [NSData]
```

Comment:
Made this non-optional and moved the initialization to `init(url: URL)`. Stylewise, would we prefer to initialize it on this line?